src/types/product.ts

```diff
@@ -0,0 +1,11 @@
+export interface Product {
+  id: string,
+  name: string,
+  imageURL: string,
+  description: string,
+  brand: string,
+  price: number
```

Comment:
if you are gonna formate big numbers i think you are gonna need to turn it to string 
e.g. 1,253